### PR TITLE
Hardware selection criteria

### PR DESCRIPTION
**Pull request by gustavosinbandera1**
please don accept this , now I am working on that 
This are the missing items:
- [ 1] Battery Power Managament System
- [ 2] Microcontroller selection
- [ 3] Hardware Block diagram
- [ 4] Control App work flow
- [ 5] Turpial General vision Block Diagram

### CODE REVIEW

**Comment by luisan00** [comment]
For a better understanding change the name **BabySitter** to **ESC** (energy  source control) or **power source & battery management **

File: README.md [hunk 1]
```diff
@@ -45,18 +56,20 @@ You can support the Locha Mesh open-source software and hardware efforts by dona
 
 <br>
 
+
+
```
Comment (jeandudey) [inline README.md]: Unrelated changes.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -45,18 +56,20 @@ You can support the Locha Mesh open-source software and hardware efforts by dona
...
 ## Getting started
 
+In this page you will find all the information related to the progress of the project, mainly in two diferents languages,[Spanish](./documentation/ES/implementacion.MD), and [English](./documentation/EN/implementation.MD). This work are still under development stage, but all day we are uodating the information to expose all the progress about the project.
```
Comment (jeandudey) [inline README.md]:
```suggestion
In this page you will find all the information related to the progress of the project, in two languages, [Spanish](./documentation/ES/implementacion.MD), and [English](./documentation/EN/implementation.MD). This work is still under development stage, but all days we are updating the information to expose all the progress about the project.
```

File: README.md [hunk 3]
```diff
@@ -96,17 +102,18 @@ Radio module, battery type, and more details about the board are currently
...
+
+<figure>
+  <img src="images/turpial-finished-board.jpg" alt="drawing" height="533" width="400" align="center" />
+  <figcaption>Turpial Board</figcaption>
+</figure>
+
+<figure>
+  <img src="images/turpial-size-compared.jpg" alt="drawing" height="533px" width="400px" align="center" />
+  <figcaption>Size comparison of a Turpial board</figcaption>
+</figure>
```
Comment (jeandudey) [inline README.md]:
```suggestion
<figure align="center">
  <img src="images/turpial-finished-board.jpg" alt="drawing" height="533" width="400" />
  <figcaption>Turpial Board</figcaption>
</figure>

<figure align="center">
  <img src="images/turpial-size-compared.jpg" alt="drawing" height="338px" width="600px" />
  <figcaption>Size comparison of a Turpial board</figcaption>
</figure>
```

This aligns the figures to the center and gives the correct size.

Comment (jeandudey) [reply README.md]: I haven't tested it

Comment (jeandudey) [reply README.md]: But the size is correct, alignment is unknown

**Comment by gustavosinbandera1** [comment]
This PR can be reviewed now, I just need to add the diagram block for android app

**Comment by luisan00** [comment]
Here we go 🛵
